pom.xml

```diff
@@ -498,7 +498,7 @@
         <groupId>org.slf4j</groupId>
         <artifactId>jcl-over-slf4j</artifactId>
         <version>${slf4j.version}</version>
-        <!-- <scope>runtime</scope> --> <!-- more correct, but scalac 2.10.3 doesn't like it -->
+        <scope>runtime</scope>
```

Comment:
@HyukjinKwon it was this change. I think we need to retain it to keep commons-logging (well, the SLF4J shim for it) on the classpath. SBT isn't including it in the build somehow even though it's `runtime` scope, which is really correct. I think we've seen this with the Maven-SBT plugin before.